articles/install-guide/index.md

```diff
@@ -17,237 +17,11 @@ Learn how to install the Microsoft Quantum Development Kit (QDK), so that you ca
-
-    You now have the Quantum Development Kit installed and ready to use in your own applications and libraries.
-
-1. Verify the installation by creating a `Hello World` application
-
-    - Create a new project:
-
-        - Go to **View** -> **Command Palette**
-        - Select **Q#: Create New Project**
-        - Navigate to the location on the file system where you would like to create the application
-        - Click on the **Open new project...** button, once the project has been created
-
-    - Run the application:
-
-        - Go to **Debug** -> **Start Without Debugging**
-        - You should see the following text in the output window `Hello quantum world!`
-
-> [!NOTE]
-> * > * Workspaces with multiple root folders are not currently supported by the Visual Studio Code extension. If you have multiple projects within one VS Code workspace, all projects need to be contained within the same root folder.
-
-## Develop with C#, using the `dotnet` command-line tool
-
-Of course, you can also build and run Q# programs from the command line by simply installing the .NET Core SDK and the QDK project templates. 
-
-1. Pre-requisites
-
-    - [.NET Core SDK 3.0 or later](https://www.microsoft.com/net/download)
-
-1. Install the Quantum project templates for .NET
-
-    ```bash
-    dotnet new -i Microsoft.Quantum.ProjectTemplates
-    ```
-
-    You now have the Quantum Development Kit installed and ready to use in your own applications and libraries.
-
-1. Verify the installation by creating a `Hello World` application
-
-    - Create a new application
-
-       ```bash
-       dotnet new console -lang Q# -o runSayHello
-       ```
-
-    - Navigate to the new application directory
-
-       ```bash
-       cd runSayHello
-       ```
-
-    You should see that two files have been created, along with the project files of the application: a Q# file (`Operation.qs`) and a C# host file (`Driver.cs`).
-
-    - Run the application
-
-        ```bash
-        dotnet run
-        ```
-
-        You should see the following output: `Hello quantum world!`
-
-## What's next?
-
-Now that you have installed the Quantum Development Kit in your preferred environment, you can write and run [your first quantum program](xref:microsoft.quantum.write-program).
+- [Q# + Python:](xref:microsoft.quantum.install.python) choose this environment if you want to combine Python and Q# to create a Python host program that calls Q# operations whenever they are needed.
+- [Q# + C#:](xref:microsoft.quantum.install.cs) choose this environment if you want to combine C# and Q# to create a C# host program that calls Q# operations whenever they are needed.
+- [Q# Jupyter Notebooks:](xref:microsoft.quantum.install.jupyter) this environment is great for writing Q# code with embedded explanations or quantum computing interactive tutorials. Do not choose this environment if you want to combine Q# with an external classical host program.
```

Comment:
```suggestion
- [Install Q# for Jupyter Notebooks:](xref:microsoft.quantum.install.jupyter) this environment is great for writing Q# code with embedded explanations or quantum computing interactive tutorials. Do not choose this environment if you want to combine Q# with an external classical host program.
```

Reply:
I prefer to introduce the concept "Q# Jupyter Notebooks" instead of "Q# for Jupyter Notebooks". This is because people (including myself) tend to link Jupyter Notebooks with Python Jupyter Notebooks. Writing "Q# for Jupyter Notebooks" leaves the door open to the reader to think that we mean Q# with Python Jupyter Notebooks while "Q# Jupyter Notebooks" leaves clear that the Jupyter Notebooks we are referring here are just for writing the Q# language.